extensions/git/src/git.ts

```diff
@@ -1172,6 +1172,11 @@ export class Repository {
 			args.push('--all');
 		}
 
+		if (options.prune) {
+			args.push('-p');
```

Comment:
I believe more understandable would be to use full argument name `--prune`

Reply:
Agree, just habit of typing the short form in the CLI 😄